### PR TITLE
PAE-438 - Header VUE theme

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
This PR adds the header with pearson-vue style

Screenshots
-----------------
**authenticated:**
image

**not authenticated:**
image

**course view:**
image

**Ticket:** [PAE-438](https://pearsonadvance.atlassian.net/browse/PAE-438)

**Reviewers:**
-----------------
- @diegomillan 
- @ivanvgh 
- @Squirrel18

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/templates/header/header.html [hunk 1]
```diff
@@ -1,37 +1,125 @@
...
+from openedx.core.djangolib.markup import HTML, Text
 
 # App that handles subdomain specific branding
 from branding import api as branding_api
+from openedx.core.djangoapps.lang_pref.api import header_language_selector_is_enabled, released_languages
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+%>
+<%
+theme = configuration_helpers.get_value('theme', '')
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/header/header.html]: I think this variable is no longer used on this template. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html [hunk 2]
```diff
@@ -14,15 +14,13 @@
 %>
 
 <h1 class="header-logo">
-  <a href="${branding_api.get_home_url()}">
+  <a href="${configuration_helpers.get_value('LMS_ROOT_URL', branding_api.get_home_url())}">
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html:17]: Is branding_api.get_home_url() not site-aware? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html [hunk 3]
```diff
@@ -9,29 +9,38 @@
 from django.urls import reverse
 from django.utils.translation import ugettext as _
 from six import text_type
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
 <%
+  theme = configuration_helpers.get_value('theme', '')
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html]: I think this variable is no longer used on this template. @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

**Review by ivanvgh — APPROVED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/header/navbar-logo-header.html:17]: is not site-aware

File: edx-platform/pearson-vue-theme/lms/templates/header/header.html (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/header/header.html]: removed

File: edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html (hunk 3 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html]: removed

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**